pkg/ebpf/processor_funcs.go

```diff
@@ -137,6 +138,12 @@ func (t *Tracee) processSchedProcessExec(event *trace.Event) error {
 		for _, pid := range pids { // will break on success
 			err = nil
 			sourceFilePath := fmt.Sprintf("/proc/%s/root%s", strconv.Itoa(int(pid)), filePath)
+
+			fileId := &fileUniqueId{}
+			if err := fileId.Stat(sourceFilePath); err != nil {
```

Comment:
Why don't we use the inode and device from the event?

Reply:
Yep, I'll do. https://github.com/aquasecurity/tracee/pull/3746#discussion_r1418842702